README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task-move-spider-DOM/)
+    - [DEMO LINK](https://11Оля112.github.io/js_task-move-spider-DOM/)
```

Comment:
```suggestion
    - [DEMO LINK](https://11Olya112.github.io/js_task-move-spider-DOM/)
```

Reply:
❗ check it

Reply:
![image](https://user-images.githubusercontent.com/98876779/183046195-0f75bef9-9e04-4bc7-81cf-f882b94d6dc3.png)